web/src/components/RemoteContentSandBox.ts

```diff
@@ -15,6 +15,7 @@ const RemoteContentSandBox = styled.div<{ $centered: boolean; $smallText: boolea
     img {
     max-width: 100%;
     max-height: 100%;
+    filter: ${props => props.theme.isContrastTheme && 'invert(1)'};
```

Comment:
❓ I am actually not sure if we should do this for all images. I think its okay that standard images have not that good contrast but it would feel weird for them to be inverted imo.

Reply:
I think it implies on all the possible affected formats of image

Reply:
I think we should discuss this in integreat-general then 🤔 But the best option would probably to only use svgs for icons and not apply this for anything else.